### PR TITLE
[AR] week03-03

### PR DESCRIPTION
**Pull request by ElJAZRY**

### CODE REVIEW

**Review by AliEl-01 — COMMENTED** [review]
actually I don’t know why you don’t delete the English text. Personally, when I check and confirm the translation, I delete the English text. Please, if there is a valid reason for that tell me 🙏

**Comment by ElJAZRY** [comment]
No, you should keep the English text commented  in the same file, don't delete it.

**Review by Atcold — APPROVED** [review]
Thanks!

**Comment by ElJAZRY** [comment]
[AR] week03-03

**Comment by ElJAZRY** [comment]
[AR] week03-03

File: docs/ar/week03/03-2.md [hunk 1]
```diff
@@ -77,8 +77,12 @@ translator: Mohamed Adel Musallam
 
 <center>
     <img src="{{site.baseurl}}/images/week03/03-2/detailed_convNet.png" width="600px" /><br>
+<<<<<<< HEAD
+    <b>شكل 1.</b> معمارية الشبكات الالتفافية
+=======
 
     <b>شكل 1.</b> معمارية الشبكات الالتفافية  
+>>>>>>> 86fa5454e8420df3a360b1cba1671ec8641ae2ab
```
Comment (Atcold) [inline docs/ar/week03/03-2.md:85]: What is this supposed to be?

File: docs/ar/week03/03-3.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
+<!-----
+lang-ref: ch.03-3
+title: Properties of natural signals
+lecturer: Alfredo Canziani
+authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
+translation data: 27 Aug 2020
+translator: Mohamed Adel Musallam
+----->
```
Comment (Atcold) [inline docs/ar/week03/03-3.md:8]: Header is missing.

File: docs/ar/week03/03-3.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
+<!-----
+lang-ref: ch.03-3
+title: Properties of natural signals
```
Comment (Atcold) [inline docs/ar/week03/03-3.md:3]: The title is not translated.

File: docs/ar/week03/03-3.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+lecturer: Alfredo Canziani
+authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
+translation data: 27 Aug 2020
```
Comment (Atcold) [inline docs/ar/week03/03-3.md:6]: `translation-date` without space and with a dash.

File: docs/ar/week03/03-3.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+translator: Mohamed Adel Musallam
+----->
+
+
+<!--
+## [Properties of natural signals](https://www.youtube.com/watch?v=kwPWpVverkw&t=26s)
+-->
+
+<div dir='rtl'>
```
Comment (Atcold) [inline docs/ar/week03/03-3.md]: Remove.

File: docs/ar/week03/03-3.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+<center> <img src = "{{site.baseurl}} / images / week03 / 03-3 / الشكل 9 مثيلات من MNIST Dataset.png" width = "350px" /> <br>
+<b> الشكل 9: </ b> مثيلات من مجموعة بيانات MNIST المتغيرة </center>
+
+<!--The performance of the FC network almost stayed unchanged ($85\%$), but the accuracy of CNN dropped to $83\%$. This is because, after a random permutation, the images no longer hold the three properties of locality, stationarity, and compositionality, that are exploitable by a CNN.
+-->
+
+ظل أداء شبكة FC دون تغيير تقريبًا ($85\%$) ، لكن دقة CNN انخفضت إلى $83\%$. هذا لأنه ، بعد التقليب العشوائي ، لم تعد الصور تحمل الخصائص الثلاثة للموقع ، والثبات ، والتكوين ، التي يمكن لشبكة CNN استغلالها.
+
+
+</div>
```
Comment (Atcold) [inline docs/ar/week03/03-3.md]: Remove.

**Review by Atcold — COMMENTED** [review]
Thanks. A few edits, as shown below, are necessary.
Also, you need to update `docs/_config.yml`.

File: docs/ar/week03/03-2.md [hunk 7]
```diff
@@ -77,6 +77,9 @@ translator: Mohamed Adel Musallam
 
 <center>
     <img src="{{site.baseurl}}/images/week03/03-2/detailed_convNet.png" width="600px" /><br>
+<<<<<<< HEAD
+    <b>شكل 1.</b> معمارية الشبكات الالتفافية
+=======
```
Comment (Atcold) [inline docs/ar/week03/03-2.md:82]: What is this?

File: docs/ar/week03/03-3.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
+<!-----
```
Comment (Atcold) [inline docs/ar/week03/03-3.md:1]: What is this?

File: docs/ar/week03/03-3.md [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+lang-ref: ch.03-3
+title: خصائص الإشارات الطبيعية
+lecturer: Alfredo Canziani
+authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
+date: 11 Feb 2020
+lang: ar
+translation-data: 27 Aug 2020
+translator: Mohamed Adel Musallam
+----->
```
Comment (Atcold) [inline docs/ar/week03/03-3.md:10]: These are supposed to be three dashes `---`.

**Review by Atcold — CHANGES_REQUESTED** [review]
Please, fix the following issues.
Thanks!

**Review by Atcold — APPROVED** [review]
Great! Thanks!

**Pull request merged**